components/PostForm/PostForm.js

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+
+export function Postform() {
+  function handleClick(event) {
```

Comment:
initialise the state here:
```suggestion
export function Postform() {
  const [value, setValue] = useState("");

  function handleClick(event) {
```

(don't forget to import)